tests/test_rpc.py

```diff
@@ -1348,3 +1348,317 @@ def test_gethistory(revault_network, bitcoind, executor):
+    CSV = 12
+    revault_network.deploy(
+        2,
+        1,
+        csv=CSV,
+        bitcoind_rpc_mocks={"estimatesmartfee": {"feerate": 0.0005}},  # 50 sats/vbyte
+    )
+    man = revault_network.mans()[0]
+    vaults = revault_network.fundmany([1, 2, 3])
+
+    # Broadcast the unvaults and get their txids
+    revault_network.activate_fresh_vaults(vaults)
+    revault_network.broadcast_unvaults_anyhow(vaults, priority=True)
+
+    unvault_txids = get_unvault_txids(man, vaults)
+    for w in revault_network.participants():
+        wait_for(
+            lambda: len(w.rpc.listvaults(["unvaulting"])["vaults"]) == len(vaults),
+        )
+
+    # If the feerate isn't significantly lower than the estimate, we won't feebump.
+    # Note the Unvault txs have a fixed 24sat/vb feerate.
+    entry = bitcoind.rpc.getmempoolentry(unvault_txids[0])
+    assert int(entry["fees"]["base"] * COIN / entry["vsize"]) == 24
+    revault_network.bitcoind_proxy.mocks["estimatesmartfee"] = {
+        "feerate": 26 * 1_000 / COIN
+    }
+    bitcoind.generate_blocks_censor(1, unvault_txids)
+    man.wait_for_logs(["Checking if transactions need CPFP...", "Nothing to CPFP"])
+
+    # The feerate is still kept the same and we will trigger CPFP manually using
+    # rpc call. This should behave in the same way as an automatic CPFP trigger
+    man.rpc.cpfp([unvault_txids[0]], 50)
+    man.wait_for_log("CPFPed transactions")
+    wait_for(lambda: len(bitcoind.rpc.getrawmempool()) == len(unvault_txids) + 1)
+
+    entry = bitcoind.rpc.getmempoolentry(unvault_txids[0])
+    assert entry["descendantcount"] == 2
+    package_feerate = entry["fees"]["descendant"] * COIN / entry["descendantsize"]
+    assert package_feerate >= 50
+
+
+## Spend transactions
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_manual_cpfp_single_spend(revault_network, bitcoind):
+    CSV = 12
+    revault_network.deploy(
+        2,
+        1,
+        csv=CSV,
+        bitcoind_rpc_mocks={"estimatesmartfee": {"feerate": 0.0005}},  # 50 sats/vbyte
+    )
+    man = revault_network.mans()[0]
+    vaults = revault_network.fundmany([1])
+
+    # Broadcast the unvaults and get their txids
+    revault_network.activate_fresh_vaults(vaults)
+    spend_psbt = revault_network.broadcast_unvaults_anyhow(vaults, priority=True)
+
+    unvault_txids = get_unvault_txids(man, vaults)
+    spend_txid = spend_psbt.tx.hash
+
+    # Confirming the unvaults
+    bitcoind.generate_block(1, wait_for_mempool=unvault_txids)
+    for w in revault_network.participants():
+        wait_for(
+            lambda: len(w.rpc.listvaults(["unvaulted"])["vaults"]) == len(vaults),
+        )
+
+    bitcoind.generate_block(CSV - 1)
+    man.wait_for_log(f"Succesfully broadcasted Spend tx '{spend_txid}'")
+
+    for w in revault_network.participants():
+        wait_for(
+            lambda: len(w.rpc.listvaults(["spending"])["vaults"]) == len(vaults),
+        )
```

Comment:
Instead of doing all this manually you should just use the corresponding "spend vaults anyhow" helper.